roles/nf-core/defaults/main.yml

```diff
@@ -44,7 +45,7 @@ pipelines:
           accession: NC_001416.1
           sha1: 2b03cbf38a8e6816914b172f048a2fe896317df7
   - name: ampliseq
-    release: 2.5.0
+    release: 2.12.0
```

Comment:
Just for me to understand: Why are some pipeline version hardcoded here and some reference and external tag from `host_vars/deploy/main.yml` ? (`e.g. {{ rnaseq_tag }}`)

Reply:
Good question :) Both sarek and rnaseq are used in the ngi_pipeline and we need the version number to construct the path where these pipelines are located. Basically, variables are used for these two pipelines to avoid writing the version number in two different places, while for all the other pipelines the version number is used only once and doesn't strictly need a variable.

Reply:
Great. Thanks for clarification!